podcasts/DownloadsViewController+Swipe.swift

```diff
@@ -46,6 +47,12 @@ extension DownloadsViewController: SwipeTableViewCellDelegate, SwipeHandler {
     }
 
     func actionPerformed(willBeRemoved: Bool) {
+        if FeatureFlag.playlistsRebranding.enabled {
```

Comment:
@bjtitus the behaviour you pointed it's actually happening in PROD as well. For now I decided to reload the list only if the `willBeRemoved` is `true`. This should avoid the cell to close on each tap. Though I kept the old behaviour if the Playlist rebranding is off